src/components/SingleFileInput/SingleFileInput.tsx

```diff
@@ -233,6 +237,9 @@ export const SingleFileInput = ({
                   <Typography display="inline" variant="body2">
                     {dropzoneLabel}
                   </Typography>
+                  <Button sx={{ margin: "0 10px" }} variant="contained">
```

Comment:
@SarahDonvito this two block of code (STATUS.IDLE and STATUS.ERROR) are almost identical, except for `color` prop.
Try to improve code readability and reusability